src/App.js

```diff
@@ -1,24 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Books from './components/Books';
+import Categories from './components/Categories';
+import Navbar from './components/Navbar';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/Categories" element={<Categories />} />
+      </Routes>
+    </Router>
   );
```

Comment:
- [ ] Please move this to index.js to make the code better.

Reply:
Hi @arslanbisharat! Thank you for your feedback.
I am working on a project that has the same hierarchy as this one here. Do you think it was worth requesting changes?

Reply:
> Hi @arslanbisharat! Thank you for your feedback. I am working on a project that has the same hierarchy as this one here. Do you think it was worth requesting changes?

Hi @Elerqsousy, I think this project structure looks great, Good job 💪🏻! 
What @arslanbisharat suggested is something I would do too because in my opinion the routes can be accessed quicker in the index since its the parent - but it all comes down to preference 😍

Reply:
Hi @Cathella, Thank you for taking the time to write this explanation.

Reply:
Thanks, @Cathella  For explaining this to me.